packages/@aws-cdk/aws-codepipeline/lib/pipeline.ts

```diff
@@ -43,7 +43,7 @@ export interface PipelineProps {
  *
  * // ... add more stages
  */
-export class Pipeline extends cdk.Construct implements events.IEventRuleTarget {
+export class Pipeline extends cdk.Construct implements actions.IPipeline {
```

Comment:
the namespace `actions` is weird. I guess you can all this `api` (😉)

Reply:
`cpapi`?